tools/plugins/network/src/main/kotlin/net/corda/cli/plugins/network/MemberList.kt

```diff
@@ -0,0 +1,90 @@
+package net.corda.cli.plugins.network
+
+import net.corda.cli.plugins.common.RestClientUtils.createRestClient
+import net.corda.cli.plugins.common.RestCommand
+import net.corda.membership.rest.v1.MemberLookupRestResource
+import net.corda.membership.rest.v1.types.response.RestMemberInfo
+import picocli.CommandLine
+
+@CommandLine.Command(name = "members", description = ["List members"])
+class MemberList : RestCommand(), Runnable {
```

Comment:
Can we make the arguments of `getMembersList()` class variables, so `getMembersList()` is a regular private method. For example -
```suggestion
class MemberList : RestCommand(), Runnable {

    @CommandLine.Option(
        names = ["-h", "--holding-identity-short-hash"],
        arity = "1",
        description = ["Short hash of the holding identity to be checked."]
    )
    var holdingIdentityShortHash: String? = null

    @CommandLine.Option(
        names = ["-cn"],
        arity = "0..1",
        description = ["Optional. Common Name (CN) attribute of the X.500 name to filter members by."]
    )
    var commonName: String? = null
```